test/Table_Tests/src/Common_Table_Operations/Main.enso

```diff
@@ -110,6 +110,9 @@ type Test_Selection
        - is_nan_and_nothing_distinct: Specifies if the backend is able to
          distinguish between a decimal NaN value and a missing value (Enso's
          Nothing, or SQL's NULL). If `False`, NaN is treated as a NULL.
+       - nan_treated_as_greater_than_all_numbers: Specifies if NaN value is
```

Comment:
Considering renaming this to `is_nan_comparable` instead.

Reply:
yes like that.